modules/python3-matplotlib.yaml

```diff
@@ -0,0 +1,99 @@
+   sha256: 0a3c7dfbda6da1e8fce29232e8e96d987ababbbf71ebc8c75659e4132c367014
+ - type: file
+   url: http://www.qhull.org/download/qhull-2020-src-8.0.2.tgz
+   sha256: b5c2d7eb833278881b952c8a52d20179eab87766b00b865000469a45c1838b7e
+modules:
+- name: python3-numpy
+  buildsystem: simple
+  build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links=file://${PWD} --prefix=${FLATPAK_DEST} numpy --no-build-isolation
+  cleanup:
+      - /bin
+      - /lib/python3.9/site-packages/numpy/tests
+      - /lib/python3.9/site-packages/numpy/*/tests
+  sources:
+    - type: file
+      url: https://files.pythonhosted.org/packages/2c/d4/590ae7df5044465cc9fa2db152ae12468694d62d952b1528ecff328ef7fc/numpy-1.24.3.tar.gz
+      sha256: ab344f1bf21f140adab8e47fdbc7c35a477dc01408791f8ba00d018dd0bc5155
+- name: python3-importlib-resources
+  buildsystem: simple
+  build-commands:
+  - pip3 install --verbose --exists-action=i --no-index --find-links=file://${PWD} --prefix=${FLATPAK_DEST} importlib-resources --no-build-isolation
+  sources:
+    - type: file
+      url: https://files.pythonhosted.org/packages/38/71/c13ea695a4393639830bf96baea956538ba7a9d06fcce7cef10bfff20f72/importlib_resources-5.12.0-py3-none-any.whl
+      sha256: 7b1deeebbf351c7578e09bf2f63fa2ce8b5ffec296e0d349139d43cca061a81a
+    - type: file
+      url: https://files.pythonhosted.org/packages/5b/fa/c9e82bbe1af6266adf08afb563905eb87cab83fde00a0a08963510621047/zipp-3.15.0-py3-none-any.whl
+      sha256: 48904fc76a60e542af151aded95726c1a5c34ed43ab4134b597665c86d7ad556
+- name: python3-cycler
+  buildsystem: simple
+  build-commands:
+  - pip3 install --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} cycler
+  sources:
+    - type: file
+      url: https://files.pythonhosted.org/packages/5c/f9/695d6bedebd747e5eb0fe8fad57b72fdf25411273a39791cde838d5a8f51/cycler-0.11.0-py3-none-any.whl
+      sha256: 3a27e95f763a428a739d2add979fa7494c912a32c17c4c38c4d5f082cad165a3
+- name: python3-fonttools
+  buildsystem: simple
+  build-commands:
+  - pip3 install --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} fonttools
+  sources:
+    - type: file
+      url: https://files.pythonhosted.org/packages/ad/5f/20da4f41e33e77723b0100ded6539529bd159319ed49d6459a4647cdc7ee/fonttools-4.39.4-py3-none-any.whl
+      sha256: 106caf6167c4597556b31a8d9175a3fdc0356fdcd70ab19973c3b0d4c893c461
+
+- name: python3-kiwisolver
+  buildsystem: simple
+  build-commands:
+  - pip3 install --verbose --exists-action=i --no-index --find-links=file://${PWD} --prefix=${FLATPAK_DEST} cppy --no-build-isolation
+  - pip3 install --verbose --exists-action=i --no-index --find-links=file://${PWD} --prefix=${FLATPAK_DEST} kiwisolver --no-build-isolation
+  sources:
+  - type: file
+    url: https://files.pythonhosted.org/packages/a4/36/c414d75be311ce97ef7248edcc4fc05afae2998641bf6b592d43a9dee581/kiwisolver-1.4.4-cp39-cp39-manylinux_2_12_x86_64.manylinux2010_x86_64.whl
```

Comment:
how will that work on aarch64?

Reply:
ok, does it have to work on aarch64 to be accepted on flathub or should I somehow exclude aarch64 in the manifest?

Reply:
Ideally, it should be made so it works for aarch64, especially as it's an open source project.

Reply:
https://pypi.org/project/kiwisolver/#files lists wheels for aarch64. You could add two URLs here for each arch, and limit it to specific architecture with `only-arches` array

Reply:
also this look like wheels for python 3.9 but this runtime is on 3.10

this manifest seems to have the right one:
https://github.com/flathub/com.leinardi.gwe/blob/3eaa0281156c2d3bf00c9e2ef9c3b89a07d4e8f1/pypi-dependencies.json#L68